IntroToCSharp/Pages/Lessons/Collections/Arrays.razor

```diff
@@ -0,0 +1,379 @@
+      IsMultiSelect="false" 
+      Question="True or False: The values in array cannot be changed after the array is created."
+      XP=30
+      GoldAward=0
+      Explanation="">
+        <CheckBoxOption Text="True" IsCorrect="false" />
+        <CheckBoxOption Text="False" IsCorrect="true" />
+    </MultipleChoiceQuestion>
+
+    <CodeBlock>
+        string[] villians = {"Red Skull" "doctor Doom" "evilbadguy12" "Thanos" "Abomination" "Ronan the Conqueror"};
+    </CodeBlock>
+
+    <MultipleChoiceQuestion IsMultiSelect="false" Question="What is wrong with the above problem?"
+        XP=30
+        GoldAward=0
+        Explanation="">
+        <CheckBoxOption Text="The array is missing commas" IsCorrect="true" />
+        <CheckBoxOption Text="The array has more than one data type" IsCorrect="false" />
+        <CheckBoxOption Text="The array needs to use [] instead of {}" IsCorrect="false" />
+        <CheckBoxOption Text="The array needs a semicolon(;) after Ronan the Conqueror" IsCorrect="false" />
+    </MultipleChoiceQuestion>
+</Section>
+
+<Section Title="Arrays without values">
+    <MudText>
+        So far in this section we've only discussed arrays whose values we know ahead of time. The issue however
+        is sometimes we don't know what is going to be inside of an array but we do know how long the array is going to be.
+        In that case we may want to set the array to the length of the array <b>before</b> we know the values!
+    </MudText>
+
+    <CodeBlock>
+        string[] threeFavoriteSuperheroes = new string[3];
+    </CodeBlock>
+
+    <MudText>
+        In the code above we create a <b>new</b> array object with the length of 3. This means that it has 3 items in the array with indexes
+        <b> 0 through 2</b>. Remeber that every array's index starts with a 0!
+    </MudText>
+
+    <Explore Title="Mystery values">
+        <MudText>Don't run the code below just yet. Read it and answer the question.</MudText>
+        <CodeBlock>
+        string[] threeFavoriteSuperheroes = new string[3];
+        Console.WriteLine(threeFavoriteSuperheroes[0]);
+        
+        int[] threeFavoriteNumbers = new int[3];
+        Console.WriteLine(threeFavoriteNumbers[0]);
+        
+        bool[] threeTrueFalse = new bool[3]
+        Console.WriteLine(threeTrueFalse[0]);
+        </CodeBlock>
+        <Question Text="What do you think the code below will display for the string array?"/>
+        <Question Text="What do you think the code below will display for the int array?"/>
+        <Question Text="What do you think the code below will display for the bool array?"/>
+        <MudText>
+            <b>What happened?</b> <br/>
+            An array that is initizalized without any values with revert to the data types
+            <b>default value</b>. For strings, this means an empty string. For ints, this means a 0.
+            And finally, for bools this means false!
+        </MudText>
+    </Explore>
+</Section>
+
```

Comment:
Add an empty section here called "Evaluating Arrays". I want to make sure we have an explanation of what is happening under the hood with arrays. We need to talk about how arrays are "reference values".